src/VisualizationTests/HelixImageComparisonTests.cs

```diff
@@ -89,32 +92,72 @@ private void RenderCurrentViewAndCompare(string testName)
             var refbitmap = BitmapFromSource(refimage);
             var newImage = BitmapFromSource(bitmapsource);
 
+#if SAVEDEBUGIMAGES
+            var debugPath = GenerateTestDataPathFromTest(testName,true);
+            SaveBitMapSourceAsPNG(debugPath, bitmapsource);
+#endif
+
             compareImageColors(refbitmap, newImage);
         }
-
-        //TODO consider something like:
-        //(diferentPixelsCount / (mainImage.width* mainImage.height))*100
-        //for a percent image diff.
         private static void compareImageColors(Bitmap image1,Bitmap image2)
```

Comment:
I think it would be clearer if we can rename the parameters to `expectedImage`, `actualImage`, so it helps us identify which is which.

Reply:
cool, will do.